### PR TITLE
add NZBHydra into Sonarr documentation.

### PR DESCRIPTION
**Pull request by brettpetch**
added NZBHydra into the Sonarr docs for those having issues with the 'automatic' setup.

### CODE REVIEW

File: docs/applications/sonarrv3.mdx [hunk 1]
```diff
@@ -118,6 +118,34 @@ API: <copy and paste from jackett UI>
...
 4. Click `Test`. If all is good, click `Save`.
 
+### Add an indexer with NZBHydra
+Sonarr only supports a few indexers out of the box, but the indexers can be expanded by using the application [NZBHydra](nzbhydra.mdx). To add an indexer with NZBHydra, use the following steps.
+
+:::note
+By default, when receiving the newznab feeds from NZBHydra, it will need to be modified to be connected with the application. The default link looks like:
+
+```plaintext
+https://<yourhostname.ltd>/nzbhydra/api
```
Comment (flying-sausages) [inline docs/applications/sonarrv3.mdx:128]: would `localhost` not work in these instances?

**Review by flying-sausages — COMMENTED** [review]
Thanks! Would it maybe make more sense to move this into the nzbhydra docs? Not sure what we're doing in this regard with Jackett but whatever that is, this should follow it.

File: docs/applications/sonarrv3.mdx (hunk 1 above)
Comment (brettpetch) [reply docs/applications/sonarrv3.mdx:128]: my bad. I copied the jackett instructions. I will update.

**Comment by brettpetch** [comment]
> 
> 
> Thanks! Would it maybe make more sense to move this into the nzbhydra docs? Not sure what we're doing in this regard with Jackett but whatever that is, this should follow it.

Jackett was in the Sonarr section, so I reflected that when I added the NZBHydra info.

File: docs/applications/sonarrv3.mdx (hunk 1 above)
Comment (flying-sausages) [reply docs/applications/sonarrv3.mdx:128]:
```suggestion
https://localhost/nzbhydra/api
```

Comment (flying-sausages) [reply docs/applications/sonarrv3.mdx:128]: this snuck back in ha

**Pull request merged**